boa_engine/src/vm/call_frame.rs

```diff
@@ -40,6 +40,46 @@ pub struct CallFrame {
 }
 
 impl CallFrame {
+    /// ---- `CallFrame` creation methods ----
```

Comment:
This should go before the impl, so that the documentation is on the list of methods, instead of only on the first method